charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -5,9 +5,12 @@ java:
     ccd:
       resourceGroup: ccd-shared
       secrets:
-        - data-store-api-draftStoreEncryptionSecret
-        - data-store-api-draft-key
-        - ccd-data-s2s-secret
+        - name: data-store-api-draftStoreEncryptionSecret
+          alias: data-store-api-draftStoreEncryptionSecret
```

Comment:
Not specifically related to this PR, but since we're touching it: I'm guessing there wasn't an env var found for this secret, so we could just delete this?